openstack/walkthrough-launch-instance-request.md

```diff
@@ -118,7 +118,7 @@ images, this data is called "metadata" instead.
 Wondering what image metadata keys and expected values are possible? The most
 reliable list of these things unfortunately is the source code itself, which
 you can find
-[here](https://github.com/openstack/nova/blob/stable/pike/nova/objects/image_meta.py#L225-L460).
+[here](https://github.com/openstack/nova/blob/8fdb1372/nova/objects/image_meta.py#L225-L460).
```

Comment:
I went with the stable/pike branch links because a) the stable branch code doesn't change much at all, and b) the link has stable/pike in it, making it easier for people to see what it's referring to.